Crash when uploading on structured-data branch

```
04-17 10:16:01.293 29730 29730 D AndroidRuntime: Shutting down VM
04-17 10:16:01.294 29730 29730 E AndroidRuntime: FATAL EXCEPTION: main
04-17 10:16:01.294 29730 29730 E AndroidRuntime: Process: fr.free.nrw.commons, PID: 29730
04-17 10:16:01.294 29730 29730 E AndroidRuntime: java.lang.RuntimeException: Unable to destroy activity {fr.free.nrw.commons/fr.free.nrw.commons.upload.UploadActivity}: java.lang.IllegalArgumentException: Service not registered: fr.free.nrw.commons.upload.UploadController$1@4d2e4d0
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4702)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4725)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:39)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1947)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:214)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7032)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
04-17 10:16:01.294 29730 29730 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: Service not registered: fr.free.nrw.commons.upload.UploadController$1@4d2e4d0
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.LoadedApk.forgetServiceDispatcher(LoadedApk.java:1671)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.ContextImpl.unbindService(ContextImpl.java:1781)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.content.ContextWrapper.unbindService(ContextWrapper.java:730)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at fr.free.nrw.commons.upload.UploadController.cleanup(UploadController.java:84)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at fr.free.nrw.commons.repository.UploadRemoteDataSource.cleanUp(UploadRemoteDataSource.java:90)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at fr.free.nrw.commons.repository.UploadRepository.cleanup(UploadRepository.java:76)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at fr.free.nrw.commons.upload.UploadPresenter.onDetachView(UploadPresenter.java:120)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at fr.free.nrw.commons.upload.UploadActivity.onDestroy(UploadActivity.java:475)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.Activity.performDestroy(Activity.java:7681)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1306)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4687)
04-17 10:16:01.294 29730 29730 E AndroidRuntime:        ... 11 more
04-17 10:16:01.295 29730 29730 E ACRA    : ACRA caught a RuntimeException for fr.free.nrw.commons
```

--- a/app/src/main/java/fr/free/nrw/commons/repository/UploadRemoteDataSource.java
+++ b/app/src/main/java/fr/free/nrw/commons/repository/UploadRemoteDataSource.java
@@ -87,7 +87,6 @@ public class UploadRemoteDataSource {
      * Clean up the selected categories
      */
     public void cleanUp(){
-        uploadController.cleanup();
         //This needs further refactoring, this should not be here, right now the structure wont suppoort rhis
         categoriesModel.cleanUp();
         depictModel.cleanUp();

